db/schema.rb

```diff
@@ -265,4 +303,13 @@
     t.index ["item_type", "item_id"], name: "index_versions_on_item_type_and_item_id", using: :btree
   end
 
+  add_foreign_key "frameworks_frameworks", "frameworks"
```

Comment:
of course all these foreign keys are misspelled, too

Reply:
thankfully, running the migration to change the table name automagically fixes the foreign key names too :)